.github/workflows/codeql.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - release-*
+      - fix-codeql-*
```

Comment:
Do we need this or was it just for testing purposes?

Reply:
@nolancon I strongly think this should be permanent, to test pipeline. I like it :)

Reply:
Fine by me :+1: